config.yaml

```diff
@@ -2,26 +2,51 @@
 # See LICENSE file for licensing details.
 
 options:
-  server_name:
+  enable_mjolnir:
+    type: boolean
+    default: false
+    description: |
+      Configures whether to enable Mjolnir - moderation tool for Matrix.
+      Reference: https://github.com/matrix-org/mjolnir
+  public_baseurl:
     type: string
     description: |
-      Synapse server name. Must be set to deploy the charm. Corresponds to the
-      server_name option on Synapse configuration file and sets the
-      public-facing domain of the server.
+      The public-facing base URL that clients use to access this Homeserver.
+      Defaults to https://<server_name>/. Only used if there is integration with
+      SAML integrator charm.
   report_stats:
     description: |
       Configures whether to report statistics.
     default: false
     type: boolean
-  public_baseurl:
+  server_name:
     type: string
     description: |
-      The public-facing base URL that clients use to access this Homeserver.
-      Defaults to https://<server_name>/. Only used if there is integration with
-      SAML integrator charm.
-  enable_mjolnir:
+      Synapse server name. Must be set to deploy the charm. Corresponds to the
+      server_name option on Synapse configuration file and sets the
+      public-facing domain of the server.
+  smtp_enable_tls:
     type: boolean
-    default: false
-    description: |
-      Configures whether to enable Mjolnir - moderation tool for Matrix.
-      Reference: https://github.com/matrix-org/mjolnir
+    description: If enabled, STARTTLS will be used to use an encrypted SMTP
+      connection.
+    default: true
+  smtp_host:
+    type: string
+    description: The hostname of the SMTP host used for sending emails.
+    default: ''
+  smtp_notif_from:
+    type: string
+    description: defines the "From" address to use when sending emails.
```

Comment:
How about "Address for outgoing email"?

Reply:
I copied from the documentation to keep the same idea.
https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html